tests/Test.ADAL.NET.Unit/PromptBehaviorTests.cs

```diff
@@ -0,0 +1,320 @@
+﻿//----------------------------------------------------------------------
```

Comment:
these should move to integration tests project

Reply:
Done.

Reply:
I still see this in tests/Test.ADAL.NET.Unit/PromptBehaviorTests.cs